blockindex/nativestaking/stakingpb/staking.proto

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2025 IoTeX
+// This source code is provided 'as is' and no warranties are given as to title or non-infringement, merchantability
+// or fitness for purpose and, to the extent permitted by law, all liability for your use of the code is disclaimed.
+// This source code is governed by Apache License 2.0 that can be found in the LICENSE file.
+
+// To compile the proto, run:
+//      protoc --go_out=. --go-grpc_out=. *.proto
+syntax = "proto3";
+package indexpb;
+option go_package = "github.com/iotexproject/iotex-core/blockindex/stakingpb";
+
+message bucketList {
+    uint64 maxBucket = 1;
+    repeated uint64 deleted  = 2;
+}
```

Comment:
```
bucketList {
    100
    []uint64{2, 3, 18}
}
```
means current max bucketID = 100, and bucket 2, 3, 18 are deleted, this is more efficient than storing all buckets